### PR TITLE
Add flto to packages to be built

### PR DESCRIPTION
**Pull request by satmandu**
- Just added flto to these packages
- These will have to be built after mono is built on actual hardware...

### CODE REVIEW

File: packages/libwebp.rb [hunk 1]
```diff
@@ -17,16 +17,18 @@ class Libwebp < Package
...
+      ./configure #{CREW_OPTIONS} \
+      --enable-libwebpmux \
+      --enable-libwebpdemux \
+      --enable-libwebpdecoder \
+      --enable-libwebpextras \
+      --enable-sdl \
+      --enable-tiff \
+      --enable-jpeg \
+      --enable-png \
+      --enable-gif"
```
Comment (uberhacker) [inline packages/libwebp.rb:31]: No binaries?

File: packages/libgdiplus.rb [hunk 2]
```diff
@@ -20,12 +20,14 @@ class Libgdiplus < Package
...
-            --with-libgif \
-            --with-x11"
+    system  "CFLAGS='-flto=auto' CXXFLAGS='-flto=auto'
+      LDFLAGS='-flto=auto' \
+      ./configure #{CREW_OPTIONS} \
+      --with-libexif \
+      --with-libtiff \
+      --with-jpeg \
+      --with-libgif \
+      --with-x11"
```
Comment (uberhacker) [inline packages/libgdiplus.rb:30]: No binaries?

File: packages/libwebp.rb (hunk 1 above)
Comment (satmandu) [reply packages/libwebp.rb:31]: These are dependent upon mono, and I can't build mono due to the mono build requiring NOT being built in a container.

File: packages/libgdiplus.rb (hunk 2 above)
Comment (satmandu) [reply packages/libgdiplus.rb:30]: These are dependent upon mono, and I can't build mono due to the mono build requiring NOT being built in a container.

**Pull request merged**